numpy/lib/function_base.py

```diff
@@ -2159,6 +2159,7 @@ def unwrap(p, discont=pi, axis=-1):
     dd = diff(p, axis=axis)
     slice1 = [slice(None, None)]*nd     # full slices
     slice1[axis] = slice(1, None)
+    slice1 = tuple(slice1)
```

Comment:
Could just be
```
slice1 = (slice(None),) * axis + (slice(1, None),)
```
I'm starting to wonder if we shouldn't have a small helper function `slice_along_axis`...